### DISCUSSION TITLE
非project目录构建提示警告

### DISCUSSION THREAD

**Discussion by Ekkone** [xmake-io/xmake#1697] [Q&A]
在非project目录构建会提示警告

warning: You are working in the project directory(/home/ekko/work/projects/SG-xmake) and you can also
force to build in current directory via run `xmake -P .`

如何配置来关闭这个提示呢？

**Comment by waruqi** [comment]
你上层有xmake.lua，当前不是项目根目录 请到项目根目录执行xmake
